### ISSUE TITLE
Could you provide inference code in details?

### ISSUE THREAD

**Issue by rlgnswk** [Zeju1997/oft#27]
Hello, 

Thank you for your great work.

Could you provide inference code in details?

I have been training the model using your provided training script, and while training proceeds smoothly, I’m struggling with inference. When I run my own inference code, I see severe overfitting and mode collapse—output images are heavily distorted and not matching expectations.

Could you please share the detailed inference script (or notebook) you use to generate final samples?

Thanks.

**Comment by iddd45** [comment]
Hello,

Did you solve this problem?

I encountered the same question, when I run the 'eval_abalation.py' code after training model with lora in oft-db, dino metric is much lower than the paper proposed.

I'm a little curious about how the paper's metric get.

**Comment by rlgnswk** [comment]
No, I do not.
